src/test/kotlin/no/nav/syfo/testhelper/TestApiModule.kt

```diff
@@ -7,13 +7,15 @@ import no.nav.syfo.application.cache.RedisStore
 import no.nav.syfo.application.mq.MQSenderInterface
 import no.nav.syfo.brev.arbeidstaker.brukernotifikasjon.BrukernotifikasjonProducer
 import no.nav.syfo.brev.behandler.BehandlerVarselService
+import no.nav.syfo.client.altinn.AltinnClient
 import redis.clients.jedis.*
 
 fun Application.testApiModule(
     externalMockEnvironment: ExternalMockEnvironment,
     behandlerVarselService: BehandlerVarselService = mockk(),
     brukernotifikasjonProducer: BrukernotifikasjonProducer,
     mqSenderMock: MQSenderInterface,
+    altinnMock: AltinnClient = mockk(),
```

Comment:
InsertCorrespondenceV2 burde mockes ordentlig i stedet for at AltinnClient mockes med mockk.
Se f.eks Isproxy hvor dette er gjort for `IFlrReadOperations` i Fastlegeregister-oppslag.